api/nodes/serializers.py

```diff
@@ -15,15 +15,18 @@ class NodeSerializer(JSONAPISerializer):
     date_modified = ser.DateTimeField(read_only=True)
 
     links = LinksField({
```

Comment:
Interesting; the rationale behind the `LinksField` was to reduce boilerplate and avoid adding another tier in the `Serializer` class hierarchy. However, I am not sure whether `LinksField` is an improvement or not.